add-ons/istio/istio.addon.remove

```diff
@@ -0,0 +1,65 @@
+
+oc delete --ignore-not-found=true customresourcedefinitions "rules.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "attributemanifests.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "deniers.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "listcheckers.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "memquotas.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "noops.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "prometheuses.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "stackdrivers.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "statsds.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "stdios.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "svcctrls.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "checknothings.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "listentries.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "logentries.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "metrics.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "quotas.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "reportnothings.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "destinationpolicies.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "egressrules.config.istio.io"
+oc delete --ignore-not-found=true customresourcedefinitions "routerules.config.istio.io"
+
+oc delete --ignore-not-found=true configmaps "istio-mixer"
+oc delete --ignore-not-found=true services "istio-mixer"
+oc delete --ignore-not-found=true serviceaccounts "istio-mixer-service-account"
+oc delete --ignore-not-found=true deployments "istio-mixer"
+oc delete --ignore-not-found=true configmaps "istio"
+oc delete --ignore-not-found=true services "istio-pilot"
+oc delete --ignore-not-found=true serviceaccounts "istio-pilot-service-account"
+oc delete --ignore-not-found=true deployments "istio-pilot"
+oc delete --ignore-not-found=true services "istio-ingress"
+oc delete --ignore-not-found=true serviceaccounts "istio-ingress-service-account"
+oc delete --ignore-not-found=true deployments "istio-ingress"
+oc delete --ignore-not-found=true services "istio-egress"
+oc delete --ignore-not-found=true serviceaccounts "istio-egress-service-account"
+oc delete --ignore-not-found=true deployments "istio-egress"
+oc delete --ignore-not-found=true serviceaccounts "istio-ca-service-account"
+oc delete --ignore-not-found=true deployments "istio-ca"
+
+oc delete --ignore-not-found=true project istio-system
+
+sleep 30
+
+oc project myproject
```

Comment:
Need extra line at the end of file as Github will shout then :)

Reply:
why?

Reply:
Looks like Git good practice 
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file